sirepo/pkcli/roles.py

```diff
@@ -46,13 +46,30 @@ def delete_roles(*args):
     delete(*args)
 
 
+def disable_user(moderator_uid, uid_or_email):
```

Comment:
This is a bit tricky. I would say put the uid_or_email first (like the other functions in roles). This was related to a discussion we had about C++ arguments in a class switching in an API and causing @moellep confusion. There's also the general idea that the first argument is what is the subject and the other args are modifiers.

The second argument is just `moderator` (not moderator uid). It would be convenient for admins to not have to remember their uid. extract the uid lookup code in _parse_args so that it can be called separately.

Reply:
[634efd8](https://github.com/radiasoft/sirepo/pull/7070/commits/634efd840a1338c81fced52736fe4cfde315bb63)